shared/js/dataDriver.js

```diff
@@ -149,6 +151,12 @@ loop.DataDriver = function() {
             console.log(`Error handling EventSource.put: ${event.data}`, ex);
           }
           break;
+        case "error":
+          setTimeout(
+            this._connectToRoom.bind(this, this._roomToken),
```

Comment:
This seems to do the whole connection setup on error. Can we get an error after the connection has already opened? How did you test this?

If we are to use this code, it should just use `setTimeout(() => this._connectToRoom(this._roomToken))`